cfme/common/vm.py

```diff
@@ -323,9 +324,9 @@ def find_quadicon(
                     # We don't use provider-specific page (vm_templates_provider_branch) here
                     # as those don't list archived/orphaned VMs
                     if self.is_vm:
-                        sel.force_navigate(self.provider.instances_page_name)
+                        navigate_to(self.provider, 'Instances')
```

Comment:
This behavior is why I opted to call the destination Instances for both cloud and infrastructure, instead of mixing VM/Instance